go-controller/cmd/ovnkube/ovnkube.go

```diff
@@ -283,7 +286,10 @@ func runOvnKube(ctx *cli.Context, cancel context.CancelFunc) error {
 		// register ovnkube node specific prometheus metrics exported by the node
 		metrics.RegisterNodeMetrics()
 		start := time.Now()
-		ncm := controllerManager.NewNodeNetworkControllerManager(ovnClientset, nodeWatchFactory, node, nodeEventRecorder)
+		ncm, err := controllerManager.NewNodeNetworkControllerManager(ovnClientset, nodeWatchFactory, node, nodeEventRecorder)
+		if err != nil {
+			klog.Errorf("Failed to create ovnkube node network controller manageger: %v", err)
```

Comment:
typo on manager, you also need to return the error. We probably dont need the log message as well as the error. You can probably just do 
```
return fmt.Errorf(Failed to create ovnkube node network controller manager: %v", err))
```